ethminer/MinerAux.h

```diff
@@ -309,6 +309,22 @@ class MinerCLI
 				}
 			}
 		}
+                else if (arg == "--cuda-parallel-hash")
+                {
+                        try {
+                                m_parallelHash = stol(argv[++i]);
+                                if(m_parallelHash == 0 || m_parallelHash>8)
```

Comment:
I've seen your response about the values. Should we also enable more parallel hashes? Up to 16 or 32 for research?